admin_manual/ai/app_llm2.rst

```diff
@@ -18,10 +18,15 @@ Requirements
 
 * This app is built as an External App and thus depends on AppAPI v2.3.0 or higher
 * Nextcloud AIO is supported
-* Using GPU is currently not supported
+* We currently support NVIDIA GPUs and x86_64 CPUs
+* GPU Sizing
+
+   * A NVIDIA GPU with at least 8GB VRAM
+   * At least 12GB of system RAM
 
 * CPU Sizing
 
+   * At least 12GB of system RAM
    * The more cores you have and the more powerful the CPU the better, we recommend 10-20 cores
```

Comment:
gpt4all uses only 4 threads by default, so mentioning that would be good to make use of the extra cores.

Reply:
We switched to Llama.cpp though

Reply:
Yeath, thought about that. Will need to be backward compatible though.

Reply:
Let's keep it like this for now, and add an issue on the llm2 repo, once that is changed we can update the docs again.

Reply:
sure!